app/routes/questionnaire.py

```diff
@@ -110,8 +110,13 @@ def get_questionnaire(schema, questionnaire_store):
         progress_store=questionnaire_store.progress_store,
         list_store=questionnaire_store.list_store,
     )
+    completed_section_ids = questionnaire_store.progress_store.completed_section_ids
+    hub_required_sections_complete = all(
```

Comment:
Our other boolean vars usually begin with `is_` - in this case it could be an `are_` - `are_hub_required_sections_complete`